package.json

```diff
@@ -3,10 +3,15 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@fortawesome/fontawesome-free": "^5.15.4",
+    "@fortawesome/fontawesome-svg-core": "^1.2.36",
```

Comment:
no need to use font awesome, we already have installed the AntD framework so please use icons from @antd/icons. Please have a look at AntD documentation. https://ant.design/components/overview/